contentcuration/contentcuration/frontend/settings/pages/Account/index.vue

```diff
@@ -197,7 +197,7 @@
       completelyDeleteAccountLabel: 'Completely remove your account from Kolibri Studio',
       unableToDeleteAdminAccount: 'Unable to delete an admin account',
       handleChannelsBeforeAccount:
-        'You must delete these channels manually or invite others to edit them before you can delete your account.',
+        'You cannot delete accounts that have shared channels. You must delete these channels manually before you can delete your account.',
```

Comment:
I think we could say that they cannot delete accounts that have channels (in general), not specifically `shared channels` because even if they have not shared them, they still cannot delete the account.